router/client/src/client.rs

```diff
@@ -113,18 +117,36 @@ impl Client {
         while n_tokens < max_prefill_tokens {
             let truncate = min(max_input_length, max_prefill_tokens - n_tokens);
 
+            let mut input_chunks = Vec::new();
+            input_chunks.push(Chunk::Text(
+                "_test ".to_string().repeat(max_input_length as usize),
+            ));
```

Comment:
What about implementing `Chunk` into `InputChunk`, that would help ergonomics and is more likely to be inlined by the compiler no ?

Reply:
It complicates the string-conversion a bit, since we can theoretically encounter `InputChunk { chunk: None }`, but since we are creating the chunks in the router anyway, it's probably not a big issue.

Changed it to `From<InputChunk> for Chunk`.